Services/Seed.cs

```diff
@@ -147,51 +147,30 @@ public class Seed : ISeed
 
             foreach (var group in template.Groups)
             {
-                try
-                {
-                    await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
-                }
-                catch (Exception ex)
-                {
-                    this.log.Error($"Failed to seed default group {group.DisplayName}", () => new { group, ex.Message });
-                }
+                await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
             }
 
```

Comment:
Please add a log statement here for "Creating Seed Rules."

Reply:
Is the "*" required here for the ETag?

Reply:
Fixed

Reply:
@jillcary Yes, "*" is the ETag meaning for "match all".